packages/expo-updates/e2e/__tests__/setup/project.js

```diff
@@ -166,16 +170,24 @@ async function initAsync(workingDir, repoRoot, runtimeVersion) {
     cwd: localTemplatePath,
     stdio: 'inherit',
   });
-  const templateVersion = require(path.join(localTemplatePath, 'package.json')).version;
 
-  await spawnAsync(
-    localCliBin,
-    ['prebuild', '--template', `expo-template-bare-minimum-${templateVersion}.tgz`],
-    {
-      cwd: projectRoot,
-      stdio: 'inherit',
-    }
-  );
+  const localTemplatePathName = glob.sync(
+    path.join(projectRoot, 'expo-template-bare-minimum-*.tgz')
+  )[0];
+
+  if (!localTemplatePathName) {
+    throw new Error(`Failed to locate packed template in ${projectRoot}`);
+  }
+
+  await spawnAsync(localCliBin, ['prebuild', '--template', localTemplatePathName], {
```

Comment:
```suggestion
  await spawnAsync('npx', ['expo', 'prebuild', '--template', localTemplatePathName], {
```

Reply:
Actually cancel this code suggestion, I figured out what the real problem is. The old global Expo CLI is still being used in several places in the setup.